Login and Register Functionality

kenmalik opened the pull request:
Connect login and register pages to Firebase functions.

kenmalik committed:
Add configuration for development

kenmalik committed:
Connect login and register pages to Firebase

mgkram4 reviewed (commented):
Nice we need but also need just one more thing will send dm on discord

kenmalik committed:
Refactor login/register functions

Moved auth functions into a class and created an instance of that class
in the login and register pages. Also converted handlers to use string
parameters instead of closures.

mgkram4 reviewed (commented):
Nice work

mgkram4 committed:
Merge branch 'develop' into kenji-longid/login
